templates/snmpd.service-Debian.epp

```diff
@@ -0,0 +1,19 @@
+###
```

Comment:
If you're using a drop in file it's sufficient to write this and inherit everything else:
```
[Service]
ExecStart=
ExecStart=/usr/sbin/snmpd <%= $snmp::snmpd_options %>
```

Reply:
After the talk on IRC, that highlights that distribs move away from `/etc/default`:
https://salsa.debian.org/debian/net-snmp/merge_requests/1/diffs

I agreee to not use the method with `EnvironmentFile`